mkdocs.yml

```diff
@@ -42,6 +42,7 @@ pages:
       - Fields: logs/fields.md
       - Field Types: logs/field-types.md
       - Supported Date Formats: logs/supported-date-formats.md
+      - Mappings and Templates: logs/mappings-templates.md
```

Comment:
Elasticsearch terminology
Would this work:
Fields & Types

Ah, funny.  I see we already have a Fields page and Field Types page.  What you wrote in this new mappings-templates.md is very much about fields and field types.  Do we really need a new page?  The Fields page is super basic and empty.  It may make more sense to add this info there instead of having this new page.

Reply:
I see it that way - the `Fields` page is about the fields themselves and don't touch the types. The `Types` page describe the types that are possible to be used. I would extend them with Field Editor information, because once it starts working again it should be the way people should modify structure of the data inside their Logs Apps. 

The `Templates and Mappings` are something that is available in addition, for more "power users" let's say. I think it should be a separate section, similar to Elasticsearch documentation is built. I will add an information in those two pages about the `Templates and Mappings` though, to have a link there.

Reply:
> The Types page describe the types that are possible to be used. I would extend them with Field Editor information, because once it starts working again it should be the way people should modify structure of the data inside their Logs Apps

Yeah, but FE is not only about changing field types.  It's also about e.g. removing fields.
And if we look at https://sematext.com/docs/logs/fields/ today what do we see? 2 sentences referencing Common Schema, a list of fields, and a screenshot.  If we look at https://sematext.com/docs/tags/common-schema/#logs-tags we see those same fields are already listed there.  Should we even have this Fields page as such then?  IMHO we should:
# Replace the content of Fields page with a paragraph about log events having structure and fields -- basically the first paragraph from this PR, followed by info about Field Editor.
# From this page we'd have references to Field Types page and we would also have references to Mappings & Templates.

Reply:
OK, got it. I just changed it @otisg, have a look now.

Reply:
I think this looks about right.  And I assume FE docs you plan on adding at a later time, si?

Reply:
Yes, FE docs are in another JIRA I had for this sprint, but I will not make it. First - I will make it "great again" and then I'll describe how awesome it is ;)